src/dso_api/dynamic_api/routers.py

```diff
@@ -61,10 +62,40 @@ class DynamicAPIRootView(APIView):
 
     def get(self, request, *args, **kwargs):
         base = request.build_absolute_uri("/").rstrip("/")
+        datasets = list(Dataset.objects.filter(id__in=self.dataset_ids))
+
         return Response(
             {
                 "datasets": {
-                    id: base + reverse(f"dynamic_api:openapi-{id}") for id in self.dataset_ids
+                    ds.schema.id: {
+                        "id": ds.schema.id,
+                        "name": ds.name,
+                        "title": ds.schema.get("title", ""),
+                        "status": ds.schema.get("status", "Beschikbaar"),
+                        "description": ds.schema.get("description", ""),
```

Comment:
```suggestion
                        "description": ds.schema.description,
```

en `.status` moeten we maar snel toevoegen ;-) (license heb ik zojuist toegevoegd)

Reply:
Ik had het hier toevallig net met Guido over. Om het DRY te houden zouden we het beste de meta-data attributen van een dataset kunnen ontlenen aan het meta-schema.

Reply:
Maar, dat is uiteraard iets voor een aparte PR.